i18n/fr/docusaurus-plugin-content-docs/current/nodes/arabica-devnet.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_label : Arabica Devnet
```

Comment:
Are these translation pages automatically generated if there is no translation of the page yet?

Reply:
sorta, we manually add the pages for each new page added to the docs so that translators later can start translating on their individual translation directories.

There's a way to automate it that @Gunter038 has mentioned before.